### ISSUE TITLE
Call meeting hack ends non-critical sabotage

### ISSUE THREAD

**Issue by heavyutils** [g0aty/SickoMenu#27]
Maybe just force the meeting event for non-critical sabotages

**Comment by g0aty** [comment]
Do meetings still work for non critical sabotages?

**Comment by heavyutils** [comment]
> Do meetings still work for non critical sabotages?

no. It just ends the sabotage and pressing the Call Meeting again does nothing.

**Comment by heavyutils** [comment]
Wait actually it works as intended for CRITICAL ones, but does not work on lights, Comms, etc.

**Comment by g0aty** [comment]
Oh, that's good to know, I'll fix this in a hotfix update

**Comment by heavyutils** [comment]
Nope, dead bodies work just fine. They can be reported at all times.
